### ISSUE TITLE
Differences between the two implementations in paper [1] and voxsrc 20 summary paper

### ISSUE THREAD

**Issue by Evan0-0** [clovaai/voxceleb_trainer#88]
Hi @joonson 
Thank you so much for your contribution on this open source work, it really helps me a lot.
I have read your voxsrc 20 summary [paper](https://arxiv.org/pdf/2010.15809.pdf).The EER of model Q/ASP using AP loss function in paper [[1]](https://arxiv.org/pdf/2003.11982.pdf) is 2.05 on Vox1 test and 1.58 on Q3 model in voxsrc 20 summary [paper](https://arxiv.org/pdf/2010.15809.pdf).Both models have the same structure and loss function.
So I wonder what is the difference between these two implementations?

**Comment by joonson** [comment]
There are differences such as the input filterbank dimension (40 vs 64), weight decay and batch size (due to half precision training).

**Comment by joonson** [comment]
Also, note the difference in test lists [here](https://github.com/clovaai/voxceleb_trainer/issues/89#issuecomment-750086656)
